source/components/GlobalNavigation/utils/uuidv4.js

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
👍 
It's the same thing that we use in quizzes - would you move it to `utils` and add readme (like utils/vignette has right now) so we can re-use it?